testing/tests/DevExpress.ui.widgets.form/form.markup.tests.js

```diff
@@ -773,6 +776,197 @@ QUnit.module('Form', () => {
     });
 });
 
+QUnit.module(`"${TOGGLE_CONTROLS_PADDING_CLASS}" class`, ()=>{
+    ['dxCheckBox', 'dxSwitch', 'dxRadioGroup'].forEach(editorType => {
+        const componentName = editorType.split('dx')[1].toLowerCase();
+
+        test(`${editorType} should have class when labelLocation=top, label.alignment=default (default=left), label.visible=true (T1126956)`, function(assert) {
```

Comment:
y do we need this information about "default"?
```suggestion
        test(`${editorType} should have class when labelLocation=top, label.alignment=left, label.visible=true (T1126956)`, function(assert) {
```